board/safety/safety_subaru.h

```diff
@@ -39,20 +39,32 @@ const LongitudinalLimits SUBARU_LONG_LIMITS = {
 #define MSG_SUBARU_ES_LKAS_State         0x322
 #define MSG_SUBARU_ES_Infotainment       0x323
 
+#define MSG_SUBARU_ES_UDS_Request        0x787
+
+#define MSG_SUBARU_ES_HighBeamAssist     0x121
+#define MSG_SUBARU_ES_STATIC_1           0x22a
+#define MSG_SUBARU_ES_STATIC_2           0x325
+
 #define SUBARU_MAIN_BUS 0
 #define SUBARU_ALT_BUS  1
 #define SUBARU_CAM_BUS  2
 
-#define SUBARU_COMMON_TX_MSGS(alt_bus, lkas_msg)    \
-  {MSG_SUBARU_ES_LKAS,         SUBARU_MAIN_BUS, 8}, \
-  {MSG_SUBARU_ES_Distance,     alt_bus,         8}, \
-  {MSG_SUBARU_ES_DashStatus,   SUBARU_MAIN_BUS, 8}, \
-  {MSG_SUBARU_ES_LKAS_State,   SUBARU_MAIN_BUS, 8}, \
-  {MSG_SUBARU_ES_Infotainment, SUBARU_MAIN_BUS, 8}, \
+#define SUBARU_COMMON_TX_MSGS(alt_bus, lkas_msg)      \
+  {lkas_msg,                     SUBARU_MAIN_BUS, 8}, \
+  {MSG_SUBARU_ES_Distance,       alt_bus,         8}, \
+  {MSG_SUBARU_ES_DashStatus,     SUBARU_MAIN_BUS, 8}, \
+  {MSG_SUBARU_ES_LKAS_State,     SUBARU_MAIN_BUS, 8}, \
+  {MSG_SUBARU_ES_Infotainment,   SUBARU_MAIN_BUS, 8}, \
+
+#define SUBARU_COMMON_LONG_TX_MSGS(alt_bus)           \
+  {MSG_SUBARU_ES_Brake,          alt_bus,         8}, \
+  {MSG_SUBARU_ES_Status,         alt_bus,         8}, \
 
-#define SUBARU_COMMON_LONG_TX_MSGS(alt_bus)         \
-  {MSG_SUBARU_ES_Brake,        SUBARU_MAIN_BUS, 8}, \
-  {MSG_SUBARU_ES_Status,       SUBARU_MAIN_BUS, 8}, \
+#define SUBARU_GEN2_LONG_ADDITIONAL_TX_MSGS(alt_bus)  \
```

Comment:
any reason to specify alt bus since it's only for gen 2?

Reply:
removed